examples/ec/provider.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: terraform.core.oam.dev/v1beta1
+kind: Provider
+metadata:
+  name: ec
```

Comment:
You can set the name to `default`, so don't need to specify `spec. providerRef` field https://github.com/mattkirby/terraform-controller/blob/ae13027fb64b2ec5a87798c774875a55c5986f90/api/v1beta1/configuration_types.go#L49.

If you set the name to ec, here is an example https://github.com/oam-dev/terraform-controller/blob/master/examples/alibaba/other-provider/configuration_oss.yaml#L40-L41.

Reply:
Thanks. I updated this name to `default` however I found I needed to set the provider name explicitly for the credential to be loaded.